provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/RequestData.java

```diff
@@ -9,6 +9,7 @@
 
 import javax.net.ssl.SSLContext;
 
+@SuppressWarnings("SameParameterValue") // Some constructors currently are passed null params, but they are designed to be generic.
```

Comment:
this class has multiple constructors with weird combination of params, which is making it difficult to categorize them as per use-case; so I am suppressing this warning instead.